ports/ztd-text/vcpkg.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "ztd-text",
+  "version": "0.0.1",
```

```suggestion
  "version-date": "2022-03-20",
```
The value of the version-date should be the date you submitted this PR.